apps/judicial-system/backend/src/app/formatters/pdf.ts

```diff
@@ -327,10 +328,19 @@ export async function generateRulingPdf(existingCase: Case): Promise<string> {
     )
     .font('Helvetica')
     .fontSize(12)
-    .text(existingCase.accusedPlea, {
-      lineGap: 6,
-      paragraphGap: 0,
-    })
+    .text(
+      `${
+        existingCase.accusedPleaDecision === AccusedPleaDecision.ACCEPT
+          ? `Kærði samþykkir kröfuna. `
+          : existingCase.accusedPleaDecision === AccusedPleaDecision.REJECT
+          ? `Kærði hafnar kröfunni. `
+          : null
+      } ${existingCase.accusedPleaAnnouncement}`,
```

Comment:
Should this be on a separate line?

Reply:
This was intentional, thought it looked neat. I got no objections to this when I asked about it in slack

![Screen Shot 2021-02-10 at 15 27 10](https://user-images.githubusercontent.com/3789875/107531131-723fc800-6bb4-11eb-86f6-a92033621529.png)

Reply:
ok, nice